Add note for DoubleTree (#69)
USRSE/usrse24: pages/attend/travel.md

- _Address_: 201 Marquette Avenue NW. Albuquerque, New Mexico 87102 (0.1 miles away)
- _Phone_: 505-247-3344
- _Special Rate Group Code_: 90F

>   - The special rate is available for the evenings of October 14th, 15th, and 16th.  If you need to stay the night of Thursday, October 17th as well, you will need to either:
>   - call the hotel to either make your reservation
>   - add the night of the 17th onto an existing reservation that was created online
> If you choose a checkout date beyond October 17th online, the rate for all days in your reservation will be higher.

Reviewer: ```suggestion
  - The special rate is available for the evenings of October 14th, 15th, and 16th.  If you need to stay the night of Thursday, October 17th as well, you will need to either:
     - call the hotel to either make your reservation
     - add the night of the 17th onto an existing reservation that was created online
  - If you choose a checkout date beyond October 17th online, the rate for all days in your reservation will be higher.
```

Reviewer: Knew it... Looks weird. Sorry :-/

Author: All good! Thanks for feexing

- _Address_: 201 Marquette Avenue NW. Albuquerque, New Mexico 87102 (0.1 miles away)
- _Phone_: 505-247-3344
- _Special Rate Group Code_: 90F

>   - The special rate is available for the evenings of October 14th, 15th, and 16th.  If you need to stay the night of Thursday, October 17th as well, you will need to call the hotel to either make your reservation, or to add the night of the 17th onto a reservation you first create online.  If you choose a checkout date beyond October 17th online, the rate for all days in your reservation will be higher.

Reviewer: ```suggestion
  - The special rate is available for the evenings of October 14th, 15th, and 16th.  If you need to stay the night of Thursday, October 17th as well, you will need to either:
  - call the hotel to either make your reservation
  - add the night of the 17th onto an existing reservation that was created online
If you choose a checkout date beyond October 17th online, the rate for all days in your reservation will be higher.
```

Reviewer: Just to make it a bit more explicit.

Reviewer: (That may render poorly... _Use caution_)
